README.md

```diff
@@ -1,2 +1,9 @@
-# z-index-devtool
-A UCOSP project for building a Z-Index Devtool.
+# Context
+This project is a part of [UCOSP](http://ucosp.ca/) and is being steered by Greg Tatum ([@gregtatum](https://github.com/gregtatum/)).
+
+It is intended to be a _prototype_. Additional documentation will be added to the `docs/` directory.
+
+# Project Description
```

Comment:
nit: Sub-headers should be h2s, so this should probably be `## Project Description`